solutions/dashboard/deployment/dashboard/codacy.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: batch/v1
```

Comment:
looks like these are changes done by @Sanket292001 right?

Reply:
@Sanket292001 Added code in https://github.com/Seagate/cortx-re/tree/dashboard/solutions/dashboard/config/codacy which is not tested with Argo CD. I have added code here with Argo CD changes. 
We will be removing https://github.com/Seagate/cortx-re/tree/dashboard/solutions/dashboard/config eventually.